src/ray/common/scheduling/cluster_resource_data.h

```diff
@@ -62,7 +62,7 @@ class ResourceRequest {
   const LabelSelector &GetLabelSelector() const { return label_selector_; }
 
   void SetLabelSelector(const LabelSelector &label_selector) {
```

Comment:
```suggestion
  void SetLabelSelector(LabelSelector label_selector) {
```
needs to take by value to move

Reply:
Oh oops I see, thanks for the fix